exercise/cache/points.py

```diff
@@ -88,46 +111,69 @@ def r_augment(children: List[Dict[str, Any]]) -> None:
                     entry.update({
                         'submission_count': 0,
                         'submissions': [],
-                        'best_submission': None,
-                        'submission_date': None,
-                        'points': 0,
-                        'formatted_points': '0',
-                        'passed': entry['points_to_pass'] == 0,
                         'graded': False,
                         'unofficial': False, # TODO: this should be True, but we need to ensure nothing breaks when it's changed
                         'forced_points': False,
                         'personal_deadline': None,
                         'personal_max_submissions': None,
-                        'feedback_revealed': True,
-                        'feedback_reveal_time': None,
+                        '_staff_best_submission': None,
+                        '_staff_best_submission_date': None,
```

Comment:
Yes, this was originally called `submission_date`. I decided to rename it to `best_submission_date` because it stores the date of the submission that is currently considered "best" by whichever grading mode the exercise uses. The key was indeed only used within `CachedPoints`, and to be honest I think there's no reason to actually store it in the cache because the `best_submission` key also exists. The only reason it is stored is the delayed feedback update. In that update, to support the LAST grading mode, I added the  `is_newer` function to the top of the file, and I wanted that function to have the same signature and logic as the original `has_more_points` function. But now I think I should've instead rewritten that function to accept a `current_best_submission` parameter instead of `current_entry`, so the date of the best submission wouldn't have to be stored in the `entry` (=exercise).

Reply:
If you think it is easy, then let's remove the `best_submission_date` value from the cache. Less data to store is better.

Reply:
I fixed this in #911.